app/controllers/links_controller.rb

```diff
@@ -1,6 +1,7 @@
 class LinksController < ApplicationController
   def index
     @links = Link.send(sort_method)
+    flash[:notice] = "You sorted by #{sort_params.key("1")}" if sort_params.key("1")
```

Comment:
Line is too long. [84/80]<br>Prefer single-quoted strings inside interpolations.